integration/e2e/backend/backend.go

```diff
@@ -8,14 +8,15 @@ import (
 
 	"github.com/grafana/e2e"
 	e2e_db "github.com/grafana/e2e/db"
+
 	"github.com/grafana/tempo/cmd/tempo/app"
 	util "github.com/grafana/tempo/integration"
 	"github.com/grafana/tempo/tempodb/backend/azure"
 )
 
 const (
 	azuriteImage = "mcr.microsoft.com/azure-storage/azurite"
-	gcsImage     = "fsouza/fake-gcs-server"
+	gcsImage     = "fsouza/fake-gcs-server:1.36.3"
```

Comment:
Why are we pinning the image? If there's a compelling reason should we pin azurite as well?

Reply:
When I use the latest image (1.37) the test fails very consistently (even locally). [This version was released yesterday](https://github.com/fsouza/fake-gcs-server/releases/tag/v1.37.0) and I think there might be a bug in some of the calls we use to list the bucket.

I've added some logs here (https://github.com/grafana/tempo/pull/1341/commits/37843d678bb9ff98cf4d943e8b8e1d59b888fe66) that show the block is flushed correctly but the compactor never sees it.

Reply:
Regarding pinning the azurite image: I don't feel particular strong about either option. These apis don't change much, so it should be fairly uncommon that an update to the image breaks the tests (though it just happened of course 🙈 )

Reply:
Haha, alright. In that case its ok to pin the gcs one since its breaking I guess. We'll pin the azurite one in the future if needed.

Side note: Is it concerning that the gcs tests are breaking? I don't think so because the google cloud APIs are stable, but just checking.